spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/SpatialConnect.java

```diff
@@ -258,6 +280,13 @@ public SCCache getCache() {
         return cache;
     }
 
+    /**
+     *
+     * this is the unique identifier that is App Store compliant and used
```

Comment:
Isn't this iOS specific?  Maybe we should put that we're following the Android best practice of using a UUID to identify an installed instance of the app?  https://developer.android.com/training/articles/user-data-ids.html#working_with_instance_ids_&_guids